specification/cognitiveservices/data-plane/ContentModerator/v1.0/contentmoderatorListManagement.swagger.json

```diff
@@ -0,0 +1,1446 @@
+        "description": {
+          "description": "Status description.",
+          "type": "string"
+        },
+        "exception": {
+          "description": "Exception status.",
+          "type": "string"
+        }
+      }
+    },
+    "GetAllImageIdsResponse": {
+      "description": "Gets all image Id response properties.",
+      "properties": {
+        "contentSource": {
+          "description": "Source of thecontent.",
+          "type": "string"
+        },
+        "contentIds": {
+          "description": "Id of the contents.",
+          "type": "array",
+          "items": {
+            "type": "number"
+          }
+        },
+        "status": {
+          "description": "Get Image status.",
+          "type": "object",
+          "$ref": "#/definitions/AddGetRefreshStatusResponse"
+        },
+        "trackingId": {
+          "description": "Tracking Id.",
+          "type": "string"
+        }
+      }
+    },
+    "GetAllTermsResponse": {
+      "description": "Gets all term Id response properties.",
+      "properties": {
+        "data": {
+          "description": "Term data details.",
+          "$ref": "#/definitions/TermDataResponse"
+        },
+        "paging": {
+          "description": "Paging details.",
+          "properties": {
+            "Total": {
+              "description": "Total details.",
+              "type": "number"
+            },
+            "Limit": {
+              "description": "Limit details.",
+              "type": "number"
+            },
+            "Offset": {
+              "description": "Offset details.",
+              "type": "number"
+            },
+            "Returned": {
+              "description": "Returned text details.",
+              "type": "number"
+            }
+          }
+        }
+      }
+    },
+    "TermDataResponse": {
+      "description": "Gets all term Id response properties.",
+      "properties": {
+        "language": {
+          "description": "Language of the terms.",
+          "type": "string"
+        },
+        "terms": {
+          "description": "List of terms.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/termsInListResponse"
+          }
+        },
+        "status": {
+          "description": "Get Term Status.",
+          "type": "object",
+          "$ref": "#/definitions/AddGetRefreshStatusResponse"
+        },
+        "trackingId": {
+          "description": "Tracking Id.",
+          "type": "string"
+        }
+      }
+    },
+    "termsInListResponse": {
+      "description": "Terms in list Id passed.",
+      "properties": {
+        "term": {
+          "description": "Added term details.",
+          "type": "string"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "listIdTst": {
+      "name": "ListId",
+      "in": "path",
+      "description": "List Id of the image list.",
+      "required": true,
+      "type": "string"
+    },
+    "contentTypeHeader": {
```

Comment:
You might want an `enum` to go along with this: -- see https://github.com/Azure/autorest/blob/41c3ebf69e2903497ac32a430c27a972a03acb16/Samples/test/stream-with-content-type/stream-with-content-type.yaml#L18